src/configSchema.ts

```diff
@@ -176,8 +176,15 @@ export const VALIDATION_SCHEMA = z
 			.nullish()
 			.transform((value) => (Array.isArray(value) ? value : [])),
 		apiKey: z.string().min(24).nullish(),
-		radarr: z.array(z.string().url()).nullish(),
-		sonarr: z.array(z.string().url()).nullish(),
+		radarr: z
+			.array(z.string().url())
+			.nullish()
+			.transform((value) => (typeof value === "object" ? value : [])),
```

Comment:
`typeof null === "object"` so this might be better off with `??`
```suggestion
			.transform(value => value ?? []),
```

Reply:
I was unaware, good to know.